### DISCUSSION TITLE
Add full Url on Model.Header

### DISCUSSION THREAD

**Discussion by jsalgado10** [OrchardCMS/OrchardCore#8373] [Q&A]
I am using the @await DisplayAsync(Model.Header) line to generate a link with the post title as the text. However I want to include the full path of the document.

currently the href is set as 
``<a href="/tenantcode/document-auto-route">My document</a>``

Is it possible to add the full url as shown below
``<a href="https://somewebsite.com/tenantcode/document-auto-route">My document</a>``

**Comment by jtkech** [comment]
@jsalgado10 

You can use the url helper

    <a href="@Url.Content("~/document-auto-route")>My document</a>

If your document is a content item available from your model, e.g. `Model.ContentItem`, we also have a custom link tag helper

    <a display-for="@Model.ContentItem">My document</a>

To use a custom tag helper, you need to add the related assembly to your view imports, and reference it in your project file, see
https://docs.orchardcore.net/en/dev/docs/reference/modules/Resources/#using-the-tag-helpers

**Reply by jsalgado10** [reply]
I was able to add the base url by setting the base tag on the head section
`` <base href="@Site.BaseUrl"> ``
